src/main/java/fi/dy/masa/tweakeroo/mixin/MixinClientPlayerEntity.java

```diff
@@ -127,24 +127,25 @@ private void disableDoubleTapSprint(CallbackInfo ci)
             target = "Lnet/minecraft/client/network/ClientPlayerEntity;isGliding()Z"))
     private void onFallFlyingCheckChestSlot(CallbackInfo ci)
     {
-        if (FeatureToggle.TWEAK_AUTO_SWITCH_ELYTRA.getBooleanValue() &&
-            this.input.playerInput.jump() && this.input.playerInput.forward())
-        {
-            // PlayerEntity#checkFallFlying
-            if (!this.isOnGround() && !this.isGliding() && !this.isInFluid() && !this.isClimbing() && !this.hasStatusEffect(StatusEffects.LEVITATION))
-            {
-                if (!this.getEquippedStack(EquipmentSlot.CHEST).isOf(Items.ELYTRA) ||
-                    this.getEquippedStack(EquipmentSlot.CHEST).getDamage() > this.getEquippedStack(EquipmentSlot.CHEST).getMaxDamage() - 10)
-                {
-                    InventoryUtils.equipBestElytra(this);
-                }
-            }
-        }
-        else
-        {
-            // reset auto switch item if the feature is disabled.
-            this.autoSwitchElytraChestplate = ItemStack.EMPTY;
-        }
+        Tweakeroo.player = (ClientPlayerEntity) (Object) this;
+//        if (FeatureToggle.TWEAK_AUTO_SWITCH_ELYTRA.getBooleanValue() &&
+//            this.input.playerInput.jump() && this.input.playerInput.forward())
+//        {
+//            // PlayerEntity#checkFallFlying
```

Comment:
What this code needs is a clean way to get the `PlayerEntity#checkFallFlying` value like it used to work in 1.21 instead of doing the Player Input keys.